portfolio/src/main/webapp/script.js

```diff
@@ -79,10 +84,30 @@ function getComments() {
 
 // Creates an <li> element containing text.
 function createListElement(comment) {
-  const liElement = document.createElement('li');
-  liElement.innerHTML = comment.username + ": " + comment.content + 
-                        "<br/><i>" + comment.date + "</i>";
-  return liElement;
+  const li = document.createElement('li');
+  li.setAttribute('class', 'cmmnt');
+  const div = document.createElement('div');
+  div.setAttribute('class', 'cmmnt-content');
+
+  // Comment content contains an avatar, header, and paragraph text.
+  const img = document.createElement('img');
+  img.setAttribute('class', 'avatar');
+  img.setAttribute('src', '/images/profile.jpg');
+  img.setAttribute('width', '40');
+  img.setAttribute('height', '40');
```

Comment:
To avoid confusion it would be good to set units for the width and height. I think the default is px but I'm not sure if that's always the behavior.